numpy/fft/_pocketfft_umath.cpp

```diff
@@ -362,8 +379,14 @@ add_gufuncs(PyObject *dictionary) {
 
 static struct PyModuleDef moduledef = {
```

Comment:
Windows C was happy with the old version, but windows C++ is not...
```
../numpy/fft/_pocketfft_umath.cpp(382): error C7555: use of designated initializers requires at least '/std:c++20'
```

Reply:
Right, a bit unfortunate, but not surprising.